Projects/Presentation/Sources/Home/View/HomeViewController.swift

```diff
@@ -461,6 +462,35 @@ final class HomeViewController: BaseViewController<HomeViewModel> {
                 self?.weekView.updateAllCompletedState(allCompletedDates: allCompletedDates)
             }
             .store(in: &cancellables)
+
+        viewModel.output.updateVersionPublisher
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] updateURL in
+                guard let updateURL else { return }
+
+                let alert = UIAlertController(
+                    title: "업데이트가 필요합니다",
+                    message: "원활한 이용을 위해, 빛나길을 업데이트 해주세요!",
+                    preferredStyle: .alert
+                )
```

Comment:
```suggestion
                let alert = UIAlertController(
                    title: "업데이트가 필요합니다",
                    message: "원활한 이용을 위해, 빛나길을 업데이트 해주세요!",
                    preferredStyle: .alert)
```

요것이 컨벤션에 맞는 것 같습니두아 !!!!

Reply:
[89bd156](https://github.com/YAPP-Github/Bitnagil-iOS/pull/57/commits/89bd15646483cfe1064a852255661899737dbd9d) 에서 반영했습니다. 감사합니다!